research/picai/fl_nnunet/nnunet_client.py

```diff
@@ -580,8 +628,6 @@ def get_properties(self, config: Config) -> Dict[str, Scalar]:
         planner = ExperimentPlanner(dataset_name_or_id=self.dataset_id, plans_name="temp_plans")
         with contextlib.redirect_stdout(None):  # Prevent print statements from experiment planner
             plans = planner.plan_experiment()
-
-        plans["plans_name"] = "nnUNetPlans"  # Set plans name to default
```

Comment:
Just want to confirm we wanted to delete this line? 🙂

Reply:
Thank you, that line helps prevent unecessary preprocessing and files from being generated